SignalR_Server/Connectors/QuestionDB_Connector.cs

```diff
@@ -66,6 +67,70 @@ public IList<M_QuestionCard> PullInQuestions()
             return questionList;
         }
 
+        public IList<M_QuestionCard> PullInQuestions(int numPlayers, int numRounds)
+        {
+            IList<M_QuestionCard> questionList = new List<M_QuestionCard>();
+
+            for (int i = 1; i <= numRounds; i++)
+            {
+                IList<M_QuestionCard> roundList = GetQuestionsForRound(i);
+
+                Random rand = new Random();
+                int[] randArray = Enumerable.Range(0, roundList.Count).OrderBy(x => rand.Next()).Take(numPlayers).ToArray();
```

Comment:
Not 100% sure that using OrderBy(x => rand.Next()) will guarantee no duplicates. This may be the same as picking a random question 10 times in a row and hoping you don't get the same one twice, but again I'm not 100% sure.

If it does guarantee no duplicates, then it would have to be using a single randomizing seed, which could also be the source of our issue with randomness not being so random.

Reply:
I'm not completely sure how it's doing its randomization as far as the seed, but it should be completely random because it's actually populating an array with all possible numbers and selecting the first set number so that duplicates are impossible.